tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -6,7 +6,6 @@
 import chainerx.testing
```

Comment:
Jenkins complains this is no longer used.

Reply:
Thanks, fixed.